File: Unity/QuestOfTheRoundTable/Assets/Scripts/Objects/QuestCard.cs

namespace Card
{
    public class QuestCard : StoryCard
    {
        public int Stages { get; set; }
        public string Foe { get; set; }

        public QuestCard(string name, int stages, string foe) : base(name, StoryType.Quest)
        {
            Stages = stages;
            Foe = foe;
        }
    }
}

File: server/src/main/java/com/qotrt/cards/QuestCard.java
package com.qotrt.cards;

public class QuestCard extends StoryCard {
	private int stages;
	private String[] foe;
	
	public QuestCard(String name, int stages, String[] strings) {
		super(name, TYPE.QUEST);
		this.stages = stages;
		this.foe = strings; 
	}
	
	public int getNumStages() { return this.stages; }
	public String[] getFoe() { return this.foe; }
}